surefire-booter/src/main/java/org/apache/maven/surefire/booter/spi/LegacyMasterProcessChannelProcessorFactory.java

```diff
@@ -36,8 +36,10 @@
  * @since 3.0.0-M5
  */
 public class LegacyMasterProcessChannelProcessorFactory
-    implements MasterProcessChannelProcessorFactory
+    extends AbstractMasterProcessChannelProcessorFactory
 {
+    private static final int FLUSH_PERIOD_MILLIS = 100;
```

Comment:
would be good to have this configurable. At least Integer.getInteger(""whatever name you like", 100)

Reply:
But it does not make sense. It is not a business feature and it is hidden for the user. The only purpose is that it just works, prints the console, nothing special.

Reply:
Fact is it depends the OS and potentially the tests so having a way to change a good default (>= 150ms IMHO) is good.

Reply:
I can see a use case of having in CI this value to be set to "infinite" in order to save resources.
But adding a new knob means that we will have to maintain it.
I am leaning toward keeping an hardcoded constant, but I don't feel so strong about it.